VideoApp/VideoApp/Stores/AppSettings/VideoSize.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Copyright (C) 2021 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+enum VideoSize: String, SettingOptions {
+    case vga
+    case quarterHD
```

Comment:
We acquire either 1024x768 and crop to 960x768 or acquire 1280x720 and crop to 900x720 depending on the formats supported by the phone. Neither is actually quarter HD (960x540) but they are closer to qHD than 720p so it is reasonably accurate.

A betterment would be to actually provide the resolution that is going to be captured at this time instead of a rough estimate.